### PR TITLE
adding datetime to hosts

### PR DESCRIPTION
**Pull request by aleksasiriski**
Adding a datetime / timestamp field for hosts, mainly because it eases how my [script](https://github.com/aleksasiriski/rffmpeg-autoscaler) works (and allows for my script to be restarted and continue working afterwards)

### CODE REVIEW

**Comment by aleksasiriski** [comment]
I've not yet tested this, will do ASAP. Until then don't merge

**Comment by aleksasiriski** [comment]
Currently I won't be adding any display info in the status command to use this new feature since I'm working on a complete rewrite in Go, but I needed to add this to test my other script.

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**
